File: logica-programacao/python/012.py
"""
Verificação de paridade 

Dado um número inteiro n:

Gere o valor 1 se o número for ímpar

Gere o valor 0 se for par
"""

numero_inteiro = int(input("Digite um número: "))
verificacao = numero_inteiro % 2

if verificacao == 1:
    print(f"1")
else: 
    print(f"0")

File: logica-programacao/javascript/012.js
/*
Verificação de paridade 

Dado um número inteiro n:

Gere o valor 1 se o número for ímpar

Gere o valor 0 se for par
*/

let numeroInteiro = Number(prompt("Digite um número: "));
let verificacao = numeroInteiro % 2;

if (verificacao === 1) {
    console.log("1");
} else {
    console.log("0");
}